guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -49,21 +57,55 @@ ifdef::satellite[]
 +
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
-# {foreman-maintain} prep-{TargetVersion}-upgrade
+# {foreman-maintain} content prepare
 ----
 endif::[]
 +
-This might take some time on high latency systems.
-. Prepare your content for Pulp migration using the following command:
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+============Missing/Corrupted Content Summary================
+  WARNING: MISSING OR CORRUPTED CONTENT DETECTED
+  Corrupted or Missing Rpm: 1000/104583
+  Corrupted or missing content has been detected, you can examine the list of content in /tmp/unmigratable_content-20211025-74422-16cxfae and take action by either:
+  1. Performing a 'Verify Checksum' sync under Advanced Sync Options, let it complete, and re-running the migration
+  2. Deleting/disabling the affected repositories and running orphan cleanup (foreman-rake katello:delete_orphaned_content) and re-running the migration.
+  3. Manually correcting files on the filesystem in /var/lib/pulp/content/ and re-running the migration
+  4. Mark currently corrupted or missing content as skipped (foreman-rake katello:approve_corrupted_migration_content). This will skip migration of missing or corrupted content.
+----
++
+There are several causes of corrupted or missing content:
+
+  * A Repository synchronization or Content View publish or promotion occurred during the migration.
+  * Files on disk in `/var/lib/pulp/content/` became corrupted due to disk rot.
+  * Files in `/var/lib/pulp/content/` are missing due to past events such as disk loss with a partial restore.
+  * There is some mismatch between subsystems in {Project} such as Katello and Pulp.
+  This can happen if a repository or content view is improperly deleted by skipping steps in a paused {ProjectName} Task.
+  Running `foreman-rake katello:correct_repositories COMMIT=true` can correct this.
++
+You can review the list of corrupt or missing RPMs written to disk as part of the `{foreman-maintain} content migration-stats command`, `/tmp/unmigratable_content-20211025-74422-16cxfae20211120-2149-1j4pmfae` in the example above.
++
+If there are a large percentage of missing or corrupted RPMs, for example, more than 5-10%, {Team} recommends that you review the list generated and contact support for guidance.
```

Comment:
Not a native speaker, but to me it feels like it should be singular since the subject is `percentage`.
```suggestion
If there is a large percentage of missing or corrupted RPMs, for example, more than 5-10%, {Team} recommends that you review the list generated and contact support for guidance.
```